src/kernels/cell-execution/cellExecutionQueue.ts

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { noop } from 'rxjs';
```

Comment:
I should probably fix these.

Reply:
Yeah, rxjs is a large package, and slow to load.